contents/docs/session-replay/how-to-control-which-sessions-you-record.mdx

```diff
@@ -43,6 +43,18 @@ The client keeps a short buffer in-memory, so you'll still be able to see how th
     classes="rounded"
 />
 
+## With Event trigger conditions
+
+You can opt to only start recordings once your user emits a particular event. After the event is captured, the recording continues even after they leave the matching page.  
```

Comment:
```suggestion
Since posthog-js version 1.186.0, you can opt to only start recordings once your user emits a particular event. After the event is captured, the recording continues even after they leave the matching page.  
```